content/en/docs/contributing/submitting.md

```diff
@@ -23,9 +23,15 @@ If you haven't yet, please review the requirements for [contributors](../communi
 
 ### Signed Commits
 
-In order to help verify the authenticity of contributed code, we ask that your [commits be signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). 
-  All commits must be signed off to show that you agree to publish your changes under the current terms and licenses of the project.
+In order to help verify the authenticity of contributed code, we ask that your [commits be signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). All commits must be signed off to show that you agree to publish your changes under the current terms and licenses of the project.
   
-  Here are some notes we found helpful in configuring a local environment to automatically sign git commits:
-    - [GPG commit signature verification](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#gpg-commit-signature-verification)
-    - [Telling Git about your GPG key](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-gpg-key)
+Here are some notes we found helpful in configuring a local environment to automatically sign git commits:
+  - [GPG commit signature verification](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#gpg-commit-signature-verification)
+  - [Telling Git about your GPG key](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-gpg-key)
+
+If you forgot to sign your commits before pushing them, you can retrospectively sign all commits on your current branch by running:
```

Comment:
maybe "retroactively" instead of "retrospectively"?
```suggestion
If you forgot to sign your commits before pushing them, you can retroactively sign all commits on your current branch by running:
```